current-sms-undeliverable-users-by-broadcast-id/lib/northstar.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+/**
+ * Imports.
+ */
+const Northstar = require('@dosomething/northstar-js');
+const logger = require('winston');
+const config = require('../config/lib/northstar');
+
+/**
+ * Setup.
+ */
+let client;
+
+/**
+ * @return {Object}
+ */
+module.exports.createNewClient = function createNewClient() {
```

Comment:
What's the advantage here over `module.exports.createNewClient = function() { ...`?